index.js

```diff
@@ -0,0 +1,25 @@
+const db = require("./data/dbHelper.js");
+
+db.getDishes()
+  .then(dishes => console.log(dishes))
+  .catch(err => console.log(err));
+
+db.getDish(4)
+  .then(dish => console.log(dish))
+  .catch(err => console.log(err));
+
+const dish = { dish_name: "Fried Wombat" };
+
+db.addDish(dish)
+  .then(response => console.log(response))
+  .catch(err => console.log(err));
+
+const recipe = { recipe_name: "Spicy Wombat", dish_id: 10 };
+
+db.addRecipe(recipe)
+  .then(response => console.log(response))
+  .catch(err => console.log(err));
+
+db.getRecipes()
+  .then(recipes => console.log(recipes))
+  .catch(err => console.log(err));
```

Comment:
With each of these `.then().catch()` chains being identical (with both of their callbacks being identical minus the arbitrary parameter names), you could do all of these with a loop. For example, I _think_ this would work but I'm running out of time for this review! Let me know if it doesn't work and I'll come up with a working implementation (if you don't first).
```javascript
const db = require("./data/dbHelper.js");

const dish = { dish_name: "Fried Wombat" };
const recipe = { recipe_name: "Spicy Wombat", dish_id: 10 };

const logOne = one => console.log(one);

[
  db.getDishes(),
  db.getDish(4),
  db.addDish(dish),
  db.addRecipe(recipe),
  db.getRecipes()
].forEach(promise => promise.then(logOne).catch(logOne))
```

Reply:
That worked like a charm.  Looks a lot nicer now!